// import { hasUncaughtExceptionCaptureCallback } from "process"


// describe("First", () => {
//     it("It must be possible to add up 2 numbers", () => {
//         expect(2 + 2).toBe(4);
//     })
// })